src/java/org/stathissideris/ascii2image/graphics/BitmapRenderer.java

```diff
@@ -19,15 +19,14 @@
  */
 package org.stathissideris.ascii2image.graphics;
 
-import java.awt.BasicStroke;
-import java.awt.Canvas;
-import java.awt.Color;
-import java.awt.Graphics;
-import java.awt.Graphics2D;
-import java.awt.Image;
-import java.awt.Rectangle;
-import java.awt.RenderingHints;
-import java.awt.Stroke;
+import org.stathissideris.ascii2image.core.ConversionOptions;
+import org.stathissideris.ascii2image.core.RenderingOptions;
+import org.stathissideris.ascii2image.core.Shape3DOrderingComparator;
+import org.stathissideris.ascii2image.core.ShapeAreaComparator;
+import org.stathissideris.ascii2image.text.TextGrid;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
```

Comment:
Please keep the explicit imports from before, it will help with the transition to Clojure.

Reply:
Ok, apparently my IDE's setting is too much aggressive at formatting imports, let me fix it.